build.sbt

```diff
@@ -1,4 +1,4 @@
-name := "frontend"
+name := "support-frontend"
```

Comment:
If you change line 71
 ```riffRaffManifestProjectName := s"support:${name.value}"``` 
to 
```riffRaffManifestProjectName := s"support:frontend"```
the artifact name won't change in riff-raff which would preserve our deployment history and generally be nice.